kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -392,19 +404,14 @@
 
         this.$router.replace(this.closePanelRoute);
       },
-      // selectionMetadata(content) {
-      //   if (content.kind === ContentNodeKinds.TOPIC) {
-      //     const count = content.exercises.filter(exercise =>
-      //       Boolean(this.selectedExercises[exercise.id])
-      //     ).length;
-      //     if (count === 0) {
-      //       return '';
-      //     }
-      //     const total = content.exercises.length;
-      //     return this.$tr('total_number', { count, total });
-      //   }
-      //   return '';
-      // },
+      selectionMetadata(content) {
+        if (content.kind === ContentNodeKinds.TOPIC) {
+          const count = this.workingResourcePool.length;
```

Comment:
This is making it so that every card shows the same # of resources selected out of however many children there are. For example, the QA Channel's QTI folder has 3 resources in it, but when I select 4 resources from the CK12 folder the QTI card says "4 of 3 resources selected"

This should instead be using the `ancestor(s)` property on the items in the workingResourcePool where basically `count` should be "the number of items in the working resource pool who has this topic as an ancestor".